app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -347,15 +356,24 @@ def parse_persistent_volume(persistent_volume)
       new_result = parse_base_item(persistent_volume)
       new_result.merge!(parse_volume_source(persistent_volume.spec))
       new_result.merge!(
-        :type           => 'PersistentVolume',
-        :parent_type    => 'ManageIQ::Providers::ContainerManager',
-        :capacity       => parse_resource_list(persistent_volume.spec.capacity.to_h),
-        :access_modes   => persistent_volume.spec.accessModes.join(','),
-        :reclaim_policy => persistent_volume.spec.persistentVolumeReclaimPolicy,
-        :status_phase   => persistent_volume.status.phase,
-        :status_message => persistent_volume.status.message,
-        :status_reason  => persistent_volume.status.reason
+        :type                    => 'PersistentVolume',
+        :parent_type             => 'ManageIQ::Providers::ContainerManager',
+        :capacity                => parse_resource_list(persistent_volume.spec.capacity.to_h),
+        :access_modes            => persistent_volume.spec.accessModes.join(','),
```

Comment:
https://github.com/ManageIQ/manageiq/pull/7955 was taking care of "capacity" field, i didn't notice there was also access moeds, ill add a pr to fix it as well, might be a problem because we can't migrate, worst case ill add a pr and we will add it whenever possible.